src/app/parcels/add/formSections/PackingDateCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { CardProps, onChangeDateOrTime, errorText } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import { DesktopDatePicker } from "@mui/x-date-pickers";
+import { ErrorText } from "@/components/Form/formStyling";
+
+const PackingDateCard: React.FC<CardProps> = ({ errorSetter, fieldSetter, formErrors }) => {
+    return (
+        <GenericFormCard
+            title="Packing Date"
+            required={true}
+            text="What date is the parcel due to be packed?"
+        >
+            <>
+                <DesktopDatePicker
```

Comment:
Why not use the responsive one?

Reply:
I think the mobile bits of it are quite ugly/ not that much easier to use ngl. Overall I just prefer the desktop bits.